cmd/options.go

```diff
@@ -62,6 +63,7 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.BoolP("throw", "w", false, "throw warnings (like failed http requests) as errors")
 	flags.StringSlice("blacklist-ip", nil, "blacklist an `ip range` from being called")
 	flags.StringSlice("summary-trend-stats", nil, "define `stats` for trend metrics (response times), one or more as 'avg,p(95),...'")
+	flags.String("summary-time-unit", "", "define the time unit used to display the trend stats. Possible units are: 's', 'ms' and 'us'")
```

Comment:
Should we perhaps say "define and fix" to signal that the unit will be fixed to the specified unit rather than adapted to the magnitude of the response times as is the case with the default behavior?

Reply:
yeah I guess you're right, let's leave it as is then